cli/run.go

```diff
@@ -130,16 +136,29 @@ func (rc *RunCommand) runRun(args []string) error {
 		}
 		defer conn.Close()
 
+		escapeKeys := defaultEscapeKeys
```

Comment:
Does this feature need testcase to cover it?

Reply:
I cannot find a easy way to test it both by unit or integration test because of the interactive and tty. @ZYecho @fuweid Any idea?